iglu-registry/0.3.0/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM snowplow-docker-registry.bintray.io/snowplow/base:0.1.0
+LABEL maintainer="Snowplow Analytics Ltd. <support@snowplowanalytics.com>"
+
+# The version of the registry to download
+ENV IGLU_REGISTRY_VERSION="0.3.0"
+
+# The name of the archive to download
+ENV ARCHIVE="iglu_server_${IGLU_REGISTRY_VERSION}.zip"
+
+# Install Iglu Registry
+RUN mkdir -p /tmp/build && \
+    cd /tmp/build && \
+    wget -q http://dl.bintray.com/snowplow/snowplow-generic/${ARCHIVE} && \
+    unzip -d ${SNOWPLOW_BIN_PATH} ${ARCHIVE} && \
+    cd /tmp && \
+    rm -rf /tmp/build && \
+    mv ${SNOWPLOW_BIN_PATH}/iglu-server-* ${SNOWPLOW_BIN_PATH}/iglu_server.jar && \
+    chown -R snowplow:snowplow ${SNOWPLOW_BIN_PATH}
+
+# Install postgresql
+RUN apk update && \
+    apk add postgresql-client
+
+COPY ./scripts/super_api_key_init.sh ${SNOWPLOW_BIN_PATH}
+RUN chown -R snowplow:snowplow ${SNOWPLOW_BIN_PATH}
+RUN chmod +x ${SNOWPLOW_BIN_PATH}/super_api_key_init.sh && \
```

Comment:
no need for an additional layer here

Reply:
Right, it has already executable bit set in git index. Will remove it.

Reply:
you should try to minimize the number of `RUN` and `COPY` that's what I meant

Reply:
I see. I could chain commands.